### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by samosaara**
Adds FAQ.

### CODE REVIEW

**Comment by samosaara** [comment]
Do you guys have any topic in mind for me to add to the FAQ?

File: README.md [hunk 1]
```diff
@@ -175,6 +175,11 @@ if [[ "$TERMINOLOGY" -eq "1" ]]; then
 fi
 ```
 
+# FAQ
+- "_Can you please add support to [some] terminal emulator?_" - It depends. First, does it even have support to image backgrounds? So we _can_ implement it in the first place? If it does, feel free to open a new issue. Else switch to a different terminal emulator, or ask the developer of your current one to implement it.
+- "_OMG it doesn't work, help._" - Check if you have python 3.5+ and the latest version of your terminal emulator. Especially terminlogy, since it seems to have a bug about image backgrounds on older versions. If it still doesn't work. Search to see if anyone had the same issue as you. If you can't find it, and the bug is reproductible, please create a issue about it.
```
Comment (cclauss) [inline README.md]: > - "_OMG it doesn't work, help._" - Check if you have python 3.5+ and the latest

- "_OMG it doesn't work, help._" - In your terminal, type `python3 -V` to make sure that you have 3.5 or later installed.  Also, make sure you are using the latest`...

**Comment by LazoCoder** [comment]
There are 4 things that I've been wanting to add to the readme but haven't had the time to do so:
1. A lot of people ask about how to configure it with zsh. I wanted to add:
   ```
   echo PATH="$HOME/.Pokemon-Terminal:$"PATH"" >> ~/.zshrc
   source ~/.zshrc
   ```
   as per ticket #40. Also @charlesmilette mentions ```echo "pokemon pikachu" >> ~/.zshrc``` but since I don't use zsh I don't know what significance this has and how to put it in the readme appropriately.
2. A lot of people have had issues with Tilix (me included) where the commands are being read but the background is not changing. The solution is to adjust the transparency as per #41.
3. Many people had issues with Terminology (I also had this issue) since it easy to install a version which is not the latest. Although we shouldn't really be explaining to people how to install terminal emulators, I think this is an exception since its not so intuitive, #33 :
   ```
   sudo add-apt-repository ppa:niko2040/e19
   sudo apt-get update
   sudo apt install terminology
   ```
4. ```39:46: syntax error: Expected end of line but found identifier. (-2741)``` is another common error. Solution is at #68.

It makes the most sense to put these under the header "Solutions for Issues" (also maybe rename it to "Solutions for Common Issues") instead of FAQ.

**Comment by Danielapariona** [comment]
How can I see the pokemon list?

**Pull request closed without merging**